src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -431,6 +437,43 @@ export default class DataTable extends Vue {
     }
   }
 
+  _radioButton(headRow: boolean, rowData: DataTableRow | null = null) {
+    const checkedState = rowData && rowData.rowNumber && this.selectedRows.includes(rowData.rowId);
+
+    if (rowData || headRow) {
+      const radioButtonId =
+        rowData && typeof rowData === 'object' && rowData.rowNumber
+          ? `radio-button-${this._rowId(rowData.rowNumber)}`
+          : '';
+      const radioButtonName = `radio-buttons-${this._dataTableId}`;
+
+      if (headRow) {
+        return <div class={`${DATA_TABLE_CLASSES.CELL} ${DATA_TABLE_CLASSES.SELECTABLE}`}></div>;
+      } else {
+        return (
+          <div class={`${DATA_TABLE_CLASSES.CELL} ${DATA_TABLE_CLASSES.SELECTABLE}`}>
+            <div class={RADIO_CLASSES.RADIO}>
+              <input
+                class={RADIO_CLASSES.INPUT}
+                type="radio"
+                name={radioButtonName}
+                id={radioButtonId}
+                disabled={rowData?.selectionDisabled}
+                onChange={() => {
+                  if (rowData) this.selectRow(rowData);
+                }}
+                checked={checkedState}
+              />
+              <label class={RADIO_CLASSES.LABEL} for={radioButtonId} style="position: static;">
```

Comment:
Please, if possible add the styling properties via CSS instead of inline styling.